wiki/Beatmap_Editor/Design/en.md

```diff
@@ -72,13 +85,13 @@ All the toggles are as shown below:-
 -   Foreground
 -   HitObjects (automatically disabled)
 
-[An example of Layer Toggles effects](Media:SE_LT.jpg "wikilink")
+**Note:** HitObjects > Foreground > Passing/Failing > Background where ">" blocks that layer. Also, any new objects (not from "Compose") introduced are set to "Foreground" by default. To transfer, drag and drop to the desired layer's tab.
 
-**Note:** HitObjects &gt; Foreground &gt; Passing/Failing &gt; Background where "&gt;" blocks that layer. Also, any new objects (not from "Compose") introduced are set to "Foreground" by default. To transfer, drag and drop to the desired layer's tab.
+#### Transformation (timeframe-based)
 
-### Transformation (timeframe-based)
+These are the commands used for your object. Five of the commands have been put in; Move, Scale, Fade, Rotate and Colour. To use the Loop and Parameters, you will need to do some Storyboard Scripting to utilise them.
 
-These are the commands used for your object. Five of the commands have been put in; Move, Scale, Fade, Rotate and Colour. To use the Loop and Parameters, you will need to do some Storyboard Scripting to utilise them. [ An example of an object subjected to Move up/down and Fade In.](Media:SE_Move.jpg "wikilink")
+![An example of an object subjected to Move up/down and Fade In.](SE_Move.jpg "An example of an object subjected to Move up/down and Fade In.")
 
 To use them:-
 
```

Comment:
Line 98, is "SB object" a generalized term to say "click on the storyboard element"? or is it literally a button?
If it is a button, change this to ``Click `SB object` ``.
If it is a general term for "click on the storyboard element" then change this to `Click the object` or something